kratos/solving_strategies/convergencecriterias/displacement_criteria.h

```diff
@@ -124,93 +114,110 @@ class DisplacementCriteria : public ConvergenceCriteria< TSparseSpace, TDenseSpa
     ///@name Operators
     ///@{
 
-    /*Criterias that need to be called after getting the solution */
+    /**
+     * Compute relative and absolute error.
+     * @param rModelPart Reference to the ModelPart containing the problem.
+     * @param rDofSet Reference to the container of the problem's degrees of freedom (stored by the BuilderAndSolver)
+     * @param A System matrix (unused)
+     * @param Dx Vector of results (variations on nodal variables)
+     * @param b RHS vector (residual + reactions)
+     * @return true if convergence is achieved, false otherwise
+     */
     bool PostCriteria(
         ModelPart& rModelPart,
         DofsArrayType& rDofSet,
         const TSystemMatrixType& A,
         const TSystemVectorType& Dx,
         const TSystemVectorType& b
-    ) override
+        ) override
     {
-        if (SparseSpaceType::Size(Dx) != 0) //if we are solving for something
-        {
-            TDataType mFinalCorrectionNorm = TSparseSpace::TwoNorm(Dx);
+        const TDataType approx_zero_tolerance = std::numeric_limits<TDataType>::epsilon();
+        const SizeType size_Dx = Dx.size();
+        if (size_Dx != 0) { //if we are solving for something
+            SizeType size_solution;
+            TDataType final_correction_norm = CalculateFinalCorrectionNorm(size_solution, rDofSet, Dx);
```

Comment:
@RiccardoRossi @loumalouomega I know I am arriving a little late here, but this used the two norm before, so it returned the square root of this number.  Is this change intentional? I do not really know which should be used.

Reply:
Hi @ddiezrod you are right, @marandra mentioned me about it, and I forgot to revert it. One sec